sandbox/docker-compose.yml

```diff
@@ -233,11 +231,11 @@ services:
       - "traefik.http.routers.hornet-dashboard.entrypoints=web"
       - "traefik.http.services.hornet-dashboard.loadbalancer.server.port=8081"
     volumes:
-      - ./data/sandboxdb:/app/database
+      - ./data/sandboxdb/dashboard:/app/database
     command:
       - "--inx.address=hornet:9029"
       - "--dashboard.bindAddress=inx-dashboard:8081"
-      - "--dashboard.auth.identityFilePath=/app/database/dashboard/identity.key"
+      - "--dashboard.auth.identityFilePath=database/identity.key"
```

Comment:
I wanted to think about this already: I think we should stick to the same solution everywhere.
So I would change line 238 to: `- "--dashboard.auth.identityFilePath=database/dashboard/identity.key"`

Reply:
See comment above